dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22ComponentTesterTests.java

```diff
@@ -226,30 +241,21 @@ public void JSF22ComponentTester_TestCommandButtonOrder() throws Exception {
      * </h:commandLink>
      */
     @Test
-    @SkipForRepeat(EE10_FEATURES) // Skipped due to HTMLUnit / JavaScript Incompatabilty (New JS in RC5)
     public void JSF22ComponentTester_TestCommandLinkOrder() throws Exception {
-        try (WebClient webClient = getWebClient()) {
 
-            URL url = JSFUtils.createHttpUrl(jsfTestServer2, contextRoot, "testActionListenerOrder.xhtml");
-            HtmlPage page = (HtmlPage) webClient.getPage(url);
+        ExtendedWebDriver driver = new CustomDriver(new RemoteWebDriver(chrome.getSeleniumAddress(), new ChromeOptions().setAcceptInsecureCerts(true)));
 
-            if (page == null) {
-                Assert.fail("JSF22ComponentTester_TestDocTag.xhtml did not render properly.");
-            }
+        String url = JSFUtils.createSeleniumURLString(jsfTestServer2, contextRoot, "testActionListenerOrder.xhtml");
+        WebPage page = new WebPage(driver);
+        page.get(url);
+        page.waitForPageToLoad();
 
-            assertTrue(page.getWebResponse().getContentAsString().contains("Action-Listener order page"));
-
-            // Click the link to execute the methods and update the page
-            HtmlAnchor anchor = page.getAnchorByName("form:testLink");
-            page = anchor.click();
-
-            HtmlElement output = (HtmlElement) page.getElementById("testOutput");
+        assertTrue(page.isInPage("Action-Listener order page"));
 
-            if (!output.asText().contains("test action called")) {
-                Assert.fail("JSF22ComponentTester_TestCommandLinkOrder: test output is not correct: "
-                            + output.asText());
-            }
-        }
+        page.findElement(By.id("form:testLink")).click();
+        page.waitForCondition(driver1 -> page.isInPage("test action called")); 
+        
+        assertEquals(page.findElement(By.id("testOutput")).getText(), "test action called");
```

Comment:
Rather than wait for a default of 8 seconds, I waited for the text to appear instead and then I verified it again via an assert. 

You'll often see the waitforCondition and then assert in these conversions.